mantle/platform/qemu.go

```diff
@@ -1536,7 +1618,9 @@ func (builder *QemuBuilder) Exec() (*QemuInstance, error) {
 			// Alternative to fw_cfg, should be generally usable on all arches,
 			// especially those without fw_cfg support.
 			// See https://github.com/coreos/ignition/pull/905
-			builder.Append("-drive", fmt.Sprintf("if=none,id=ignition,format=raw,file=%s,readonly=on", builder.ConfigFile), "-device", "virtio-blk,serial=ignition,drive=ignition")
+			if err := builder.attachIgnitionConfig(); err != nil {
```

Comment:
The comment above this should be moved now.

But I wonder if instead of this, we leave it but right above it, we do something like:

```go
if err := builder.secexEncryptConfig(); err != nil {
	return nil, err
}
builder.Append(...)
```

where `builder.secexEncryptConfig()` encrypts the config and updates `builder.ConfigFile` if Secure Execution is enabled.

That way, both if-statement branches directly show the QEMU args that get added, which is nice.

I'm OK with the current approach too, but let's move that comment to where we append the drive.